<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mostrar_informes_ventas</class>
 <widget class="QWidget" name="mostrar_informes_ventas">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informes de ventas</string>
  </property>
  <widget class="QTableWidget" name="tabletWidget_informe_ventas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>691</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Por dia/mes/a√±o :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_marca">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Seleccione una marca</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alcatel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Apple</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BlackBerry</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Google</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Huawei</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kodak</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Motorola</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nokia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Samsung</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Xiaomi</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Marca:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Metodos de pago:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_pago">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Efectivo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debito</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Credito</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vendedor">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Vendedor (DNI):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_buscar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_dia"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_mes"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_anio"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
